services/summaries/summaries.py

```diff
@@ -0,0 +1,194 @@
+import os
+import re
+import sys
+import time
+import json
+import logging
+import traceback
+import emoji
+import random
+import datetime
+from arango import ArangoClient
+
+DB_USER = os.getenv("DB_USER")
+DB_PASS = os.getenv("DB_PASS")
+DB_PORT = os.getenv("DB_PORT")
+DB_NAME = os.getenv("DB_NAME")
+DB_HOST = os.getenv("DB_HOST")
+
+SCAN_TYPES = ["https", "ssl", "dkim", "spf", "dmarc"]
+CHARTS = {"mail": ["dmarc", "spf", "dkim"], "web": ["https", "ssl"]}
+
+logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+
+
+def update_scan_summaries(host=DB_HOST, name=DB_NAME, user=DB_USER, password=DB_PASS, port=DB_PORT):
+    logging.info(f"Updating scan summaries...")
+
+    # Establish DB connection
+    connection_string = f"http://{host}:{port}"
+    client = ArangoClient(hosts=connection_string)
+    db = client.db(name, username=user, password=password)
+
+    if not db.has_collection("scanSummaries"):
+        db.create_collection(
```

Comment:
This needs to be removed for the database initilization script to be the so it does not override any settings that we could possibly change.